src/components/Table.tsx

```diff
@@ -165,7 +232,16 @@ export default class Table extends React.Component<
 				<tbody>
 					{map(this.sortData(data), (row, i) => {
 						return (
-							<tr key={rowKey ? row[rowKey] : i}>
+							<tr key={rowKey ? (row[rowKey] as any) : i}>
+								{this.props.onChecked && (
+									<td>
+										<input
```

Comment:
Should/Could we use the rendtion `<Input />` here, so that if/when we implement a custom checkbox, the table will get it automatically?

Reply:
Yep could do